db/migrate/20150617184118_create_tasks.rb

```diff
@@ -1,11 +1,14 @@
 class CreateTasks < ActiveRecord::Migration
   def change
     create_table :tasks do |t|
-      t.string :name, null: false
-      t.string :description
+      t.string   :name, null: false
+      t.string   :description
       t.datetime :date_complete
+      t.integer  :person_id
```

Comment:
Oh, interesting! I see how that could cause problems in a project with multiple people setting up things simultaneously. In a project where the initial setup is happening in something like a live code or pair programming situation (or in a totally solo project like this one), is it still recommended to create a new migration?